OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/RenderedObjectMenuBuilder.java

```diff
@@ -9,22 +9,20 @@
 import androidx.annotation.Nullable;
 
 import net.osmand.CallbackWithObject;
-import net.osmand.ResultMatcher;
-import net.osmand.binary.BinaryMapIndexReader;
 import net.osmand.binary.ObfConstants;
 import net.osmand.data.Amenity;
+import net.osmand.data.LatLon;
 import net.osmand.data.QuadRect;
 import net.osmand.osm.MapPoiTypes;
 import net.osmand.osm.PoiType;
 import net.osmand.plus.OsmandApplication;
 import net.osmand.plus.activities.MapActivity;
+import net.osmand.plus.views.layers.MapSelectionHelper;
```

Comment:
Looks like Bad package name for MapSelectionHelper especially if it's used in search